book.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'item'
+class Book < Item
+  def initialize(publisher, cover_state)
+    @publisher = publisher
+    @cover_state = cover_state
+    super()
```

Comment:
You should call super with an argument " publish_date "  like this : super(publish_date)